runtime/onert/core/include/ir/NNPkg.h

```diff
@@ -70,9 +105,67 @@ class NNPkg
    */
   std::shared_ptr<Model> &model(const ModelIndex &index) { return _models.at(index); }
 
+  /**
+   * @brief Get pkg_input at index
+   *
+   * @param[in] index Index of pkg_input to be returned
+   * @return OperandDesc at index
+   */
+  OperandDesc input(uint32_t index) const { return _pkg_inputs[index]; }
+  /**
+   * @brief Get pkg_input at index
+   *
+   * @param[in] index Index of pkg_input to be returned
+   * @return OperandDesc at index
+   */
+  OperandDesc &input(uint32_t index) { return _pkg_inputs[index]; }
+  /**
+   * @brief Add input at the end
+   *
+   * @param[in] input Input OperandDesc to be pushed
+   */
+  void addInput(const OperandDesc &input) { _pkg_inputs.push_back(input); }
+
+  /**
+   * @brief Get pkg_output at index
+   *
+   * @param[in] index Index of pkg_output to be returned
+   * @return OperandDesc at index
+   */
+  OperandDesc output(uint32_t index) const { return _pkg_outputs[index]; }
```

Comment:
```suggestion
  const OperandDesc &output(uint32_t index) const { return _pkg_outputs[index]; }
```

Reply:
@ragmani I returned copy of `OperandDesc` because I met a compilation error during making this change, and `OperandDesc` is 3 uint32_t, its copy overhead would be small. But I checked there is no compilation error with your suggestion. I've updated. (including `input()`).